matching-game/src/components/cards/card.tsx

```diff
@@ -1,19 +1,25 @@
-import React from 'react'
+import React, { type MouseEventHandler } from 'react'
```

Comment:
```suggestion
import React from 'react'
```
since we're not using the `MouseEventHandler` type anymore, let's cut that from the imports.